### DISCUSSION TITLE
Code blocks not rendering properly

### DISCUSSION THREAD

**Discussion by dsbilling** [spatie/laravel-markdown#24] [General]
Using this in the controller:
```php
$html = app(\Spatie\LaravelMarkdown\MarkdownRenderer::class)->toHtml($post->body);
return view('blog.show', ['html' => $html]);
```
And this in the view:
```blade
{!! $html !!}
```
Will render this:
![image](https://user-images.githubusercontent.com/9788214/146594673-d29d9229-0a52-451c-8366-f7c887535b5d.png)

---

But using this in the controller:
```php
return view('blog.show', ['post' => $post]);
```
And this in the view:
```blade
<x-markdown>{{ $post->body }}</x-markdown>
```
Will return this:
![image](https://user-images.githubusercontent.com/9788214/146594811-97a685d4-6372-4ef4-aa13-28d7645fa36a.png)

---
Why is it like this? Or is there something I am missing?

**Comment by ClintWinter** [comment]
Use `{!! $post->body !!}` inside the markdown tag.
